operate/qa/integration-tests/src/test/java/io/camunda/operate/elasticsearch/FinishedImportingIT.java

```diff
@@ -44,7 +44,6 @@ public class FinishedImportingIT extends OperateZeebeAbstractIT {
   @Autowired protected ZeebeImporter zeebeImporter;
   @Autowired private RestHighLevelClient esClient;
   @Autowired private OperateProperties operateProperties;
-  @Autowired private ImportPositionIndex importPositionType;
```

Comment:
❓ why did we change this test? not a big deal, but I'm just curious

Reply:
my bad forgot how beans work